docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2371,7 +2379,7 @@ in the home.
   * - Middle initial 
   * - Last name
   * - Age 
-  * - DOB (stored as a string in MM/DD/YYYY format)
+  * - DOB (stored as a string in MMDDYYYY format)
```

Comment:
I have no preference, but what is the reason behind this change?

Reply:
We are trying to match the data to the actual data source, when possible. So some googling of data structure lead to this change so we can match real life.